vars/pipeline2ATX.groovy

```diff
@@ -199,22 +204,22 @@ static def resultToATXVerdict(result) {
  *
  * @param build
  *      the pipeline build
- * @param debug
- *      the log level
+ * @param appendLogs
+ *      if true, the individual logs of test steps are added in their description field
  * @return the test execution steps of the pipeline build
  */
 @NonCPS
-def getExecutionSteps(build, debug) {
+def getExecutionSteps(build, appendLogs) {
     def executionTestSteps = []
 
     // Table might not exist before execution is finished
     FlowGraphTable table = new FlowGraphTable(build.getExecution())
     table.build()
 
     // Get start of relevant pipeline steps
-    def topLevelRows = table.getRows().stream().findAll { r -> r.getTreeDepth() == 3 }.toArray()
-    for (FlowGraphTable.Row row : topLevelRows) {
-        def item = crawlRows(row, debug)
+    def Stages = table.getRows().stream().findAll { r -> r.getDisplayName() == "stage" }.toArray()
```

Comment:
I guess this finds all stages. TopLevel stages and nested ones?

Reply:
Yes, that is the case. So we get a similar view like in Jenkins having all stages as top-level execution steps.